README.md

```diff
@@ -34,6 +34,10 @@ This repository is the starting point for any information and tools you will nee
 
 ## Requirements
 
+We recommend a dedicated VM for the Bridgehead, with nothing else running on it.
+
+It may to be possible to run other apps on the same server, if they don't share common ports with the Bridgehead, and if they do not take up resources that the Bridgehead needs, like RAM. The Bridgehead may also have problems if other applications need older versions of git, Docker or curl.
```

Comment:
```suggestion
To guarantee a smooth operation of the Bridgehead, we recommend a dedicated VM for the Bridgehead, with no other applications running on it.

It may to be possible to run other apps on the same server, if they don't share common ports with the Bridgehead, and if they do not take up resources that the Bridgehead needs, like RAM. The Bridgehead may also run into issues if other applications need incompatible versions of git, Docker, curl, or other dependencies.
```